x/twap/migrate.go

```diff
@@ -1,9 +1,27 @@
 package twap
 
 import (
+	"errors"
+	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+type Migrator struct {
+	keeper Keeper
+}
+
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// Migrate1To2GeometricTwapAcc migrates all existing twap records
+// by initializing their geometric twap accumulator to zero.
+// Return nil on success, error otherwise.
+func (m Migrator) Migrate1To2GeometricTwapAcc(ctx sdk.Context) error {
+	return m.keeper.initializeGeometricTwapAcc(ctx, sdk.ZeroDec())
```

Comment:
Shouldn't this be `sdk.OneDec()`? (the identity element of multiplication)

Reply:
No, it should be zero because we are computing the geometric time-weighted average of prices as an arithmetic time-weighted average of prices in log scale.